cmd/ps.go

```diff
@@ -36,8 +36,10 @@ func newPsCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// psCmd represents the ps command
 	var psCmd = &cobra.Command{
 		Use:   "ps",
-		Short: "List the appsody containers running in the local docker environment",
-		Long:  `This command lists all stack-based containers, that are currently running in the local docker envionment.`,
+		Short: "List the Appsody containers running in the local docker environment.",
+		Long: `List all stack-based containers, that are currently running in the local docker envionment. 
+		
+Shows the following information about the currently running Appsody containers: container ID, container name, image and status.`,
```

Comment:
Yes, I agree it would be difficult to give an example in this case (there are a few other commands that have this "problem") @helenmasters what do you think?

Reply:
I'd be inclined to say "Shows the following information about the Appsody containers that are currently running:"

I also agree that an example isn't appropriate.